lib/ransack/adapters/active_record/base.rb

```diff
@@ -23,7 +23,8 @@ def ransacker(name, opts = {}, &block)
         end
 
         def ransack_alias(new_name, old_name)
-          self._ransack_aliases.store(new_name.to_s, old_name.to_s)
+          self._ransack_aliases = _ransack_aliases.merge new_name.to_s =>
```

Comment:
It's worth noting that this is exactly how the `ransacker` method already works.